core/src/main/java/org/jboss/galleon/diff/FsDiff.java

```diff
@@ -134,8 +137,6 @@ public static Map<String, Boolean> replay(FsDiff diff, Path home, MessageWriter
                         }
                     } else if (modifiedPathConflict(update)) {
                         glnew(target);
-                    } else {
```

Comment:
I am wandering if we should introduce a new kind of action SAFE_MODIFICATION, otherwise this change is still reported as a conflict [C] (when printOnlyConflicts is false) although it is not and we are not generating glnew file. When reporting the conflict we should check if the action is not a SAFE_MODIFICATION: action != SAFE_MODIFICATION && (!printOnlyConflicts || warning != null).

Reply:
Maybe I misunderstood the desired outcome.

I thought the goal was to print the warnings as they are now (without printOnlyConflicts) and skip the .glnew file on false conflicts. Or do we want to remove those from logs entirely? If we remove those from logs - should we leave non-conflicting addition/removal?

Or do you mean we should report those as modified [M] rather then conflict [C], signifying that the user's changes have been preserved but there's no need for user input (sort of like [+]/[-])?

Reply:
Advertising a conflict that is not a conflict seems strange to me, reporting with a [M] is a good idea. That would answer my concern.

Reply:
Added `Modified` [M] state to the diff outcome. Also changed `get-changes` command to use [M] instead of [C] because it detects files with user changes not conflicts with next version.